src/librustdoc/html/render/mod.rs

```diff
@@ -312,15 +312,15 @@ impl AllTypes {
         f.write_str(
             "<h1 class=\"fqn\">\
                  <span class=\"in-band\">List of all items</span>\
-                 <span class=\"out-of-band\">\
-                     <span id=\"render-detail\">\
-                         <a id=\"toggle-all-docs\" href=\"javascript:void(0)\" \
-                            title=\"collapse all docs\">\
-                             [<span class=\"inner\">&#x2212;</span>]\
-                         </a>\
-                     </span>
-                 </span>
-             </h1>",
+             </h1>\
```

Comment:
Why did you change this? With this change it looks like this:

![Screenshot from 2022-01-07 21-49-05](https://user-images.githubusercontent.com/3050060/148605576-63343dd3-aaf3-4444-acdf-da03493bbf7b.png)

Reply:
Good catch! This was a mistake from an earlier iteration. Reverted.